src/components/todo.js

```diff
@@ -0,0 +1,37 @@
+import {$todoList} from '../todoDOM.js';
+import {template} from '../template.js';
+import {api} from '../api.js';
+
+export const todo = {
+    load : async (userId) => {
+        const array = await api.loadTodoList(userId);
+        console.log(array);
+
+        let todos = [];
+        for(let i in array){
+            const newTodo = array[i].contents;
+            todos = [...todos, newTodo];
+        }
```

Comment:
이 부분은 JavaScript의 메서드를 활용하면 가독성이 더 좋아질거 같습니다!
```suggestion
        // 1
        let todos = [];
        for(let i in array){
            todos.push(array[i].contents);
        }
        // 2
        let todos = array.map((item) => item.contents);
```

Reply:
👆 JS의 Array는 유용한 메서드를 제공합니다.
https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/map

Reply:
감사합니다! 아직 array api를 쓰는거에 익숙하지 않아서 이렇게 놓치는 부분이 있네요 😅